templates/tools/dockerfile/php_common_deps.include

```diff
@@ -2,5 +2,9 @@
 RUN curl -sS https://getcomposer.org/installer | php
 RUN mv composer.phar /usr/local/bin/composer
 
+# Install valgrind for memory tests
+RUN apt-get update && apt-get install -y \
```

Comment:
nit: you need `${'\\'}` here to  get what you want in the generated docker file.